netwitness_connector.py

```diff
@@ -266,7 +273,7 @@ def _get_capture(self, param, cap_type):
                 except Exception as e:
                     return action_result.set_status(phantom.APP_ERROR, consts.NETWITNESS_INVALID_PARAM.format(message=e))
 
-                query += ' && time="{0}"="{1}"'.format(time1, time2)
+                query += ' && time="{0}"-"{1}"'.format(time1, time2)
```

Comment:
I believe this is the fix, right? 
Should we comment on why it's important to use the symbol `-` over `=`?

Reply:
Yes, this is the fix here. Actually, the '=' was incorrectly placed as the API expects the formatted query to be sent with the '-' sign in between, so it's just a syntactical correction and description for this is already present in the action verbose as mentioned below

NOTE: If start_time and end_time are included along with a query, then the time-frame will be appended to the end of the query. For example: if the query is ip.src=10.10.10.10, the start time is 2018-01-01 00:00:00, and the end time is 2018-01-01 23:59:59, then the final query would be ip.src=10.10.0.7 && time="2018-01-01 00:00:00"-"2018-01-01 23:59:59"

Reply:
I see. thanks!